### ISSUE TITLE
Null Reference Exception coming every time on call of any dialog for who skill in composer

### ISSUE THREAD

**Issue by RajdeepRay0411** [microsoft/BotFramework-Composer#5308]
## Describe the bug

Null Reference Exception coming every time on call of any dialog for who skill in the composer though Luis is triggering the correct flow according to the intent.

## Version

Bot Framework Composer (v 1.2.1)

## OS

- [ ] macOS
- [x] Windows
- [ ] Ubuntu

## To Reproduce

1. Download the solution for who skill from Github.
2. Run the application in the composer.
3. Trigger any intent(For example Whois)
4. Throwing exception "Object reference not set to an instance of an object."

## Expected behavior

A respective dialog should be initiated according to the intent triggered.

## Screenshots

![image](https://user-images.githubusercontent.com/51920866/101875348-78172e00-3bb0-11eb-96ef-1f56d12a83c2.png)

**Comment by dmvtech** [comment]
Hi @RajdeepRay0411 

Can you please link the Who skill? I'm not familiar with that one.

The suggestion I might have have at this point would be to eject the runtime and debug to see where it might be failing. Are you sure you have all the configuration info setup needed for the skill to be called?

**Comment by RajdeepRay0411** [comment]
Hi @dmvtech, hereby attaching the link for who skill: https://github.com/microsoft/botframework-components/tree/preview-build2020/skills/declarative/whoskill

**Comment by RajdeepRay0411** [comment]
Yes @dmvtech, all the configuration info required for the skill is added in appsettings.json. Still, I am facing the same issue.

**Comment by RajdeepRay0411** [comment]
Hi @dmvtech, any update on this issue?

**Comment by dmvtech** [comment]
I am investigating. I'll update when I have more information.

**Comment by dmvtech** [comment]
Hi @RajdeepRay0411 With my investigation so far, this sample doesn't seem complete.

First, I don't think this will _simply_ work without being deployed (or maybe tested locally with ngrok) in Emulator. I'm sure most of the issue is in the `SearchPerson` dialog as this is the one that implements the OAuth prompt (which, by the way, requires a [connection setting](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-authentication?view=azure-bot-service-4.0&tabs=aadv2%2Ccsharp#register-the-azure-ad-identity-provider-with-the-bot) (`connectionName`) configured in settings. Additionally, it is seemingly dependent on having an actual app (and therefore appid/apppassword configured). 

On further investigation, there are custom actions in the `SearchPerson` dialog, that are only coming from the schema. i.e. There are no code for these custom actions in the runtime.

Essentially; this is an issue with the sample/skill and not with Composer. I'll keep looking around and share what else I find.
